src/components/common/CommonForm.tsx

```diff
@@ -47,7 +47,7 @@ const Input: React.VFC<
       <StyledInput
         isInvalid={status === 'error'}
         isDisabled={status === 'validating'}
-        isSuccess={status === 'success'}
+        issuccess={(status === 'success').toString()}
```

Comment:
can extract status types

```
type Status = 'error' | 'validating' | 'success'
```
<img width="875" alt="image" src="https://github.com/urfit-tech/lodestar-app/assets/28040359/2754bcd5-1261-459c-a463-e12074b2fd65">

Reply:
Since this component is not currently using 'validating', maybe can skip adding it for now?